_plugins/dashboard.rb

```diff
@@ -43,6 +43,13 @@ def self.munge_project_data(project_data)
       FIELDS_TO_TRANSLATE.each do |from, to|
         project_data[to] = project_data[from] unless project_data[to]
       end
+      original_name = project_data['name']
+      if original_name.scan(/[A-Z]/).size > 0
+        project_data['redirect_from'] = Array.new
+        project_data['redirect_from'].push("project/#{original_name}")
```

Comment:
Two questions:

- Why not set `project_data['redirect_from']` = ["project/#{original_name}"] in one line?
- Is it expected that there'll need to be more than one value in `project_data['redirect_from']`? If so, you can use something like:

```ruby
(project_data['redirect_from'] ||= []) << "project/#{original_name}"
```